src/redux/studio.js

```diff
@@ -111,10 +120,32 @@ const getRoles = (studioId, username, token) => (dispatch => {
     });
 });
 
+// Selectors
+
+// Fine-grain selector helpers - not exported, use the higher level selectors below
+const isCreator = state =>
+    state.studio.owner !== null && // Never try matching if owner has not been set
+    selectUserId(state) === state.studio.owner;
+const isCurator = state => state.studio.curator;
+const isManager = state => state.studio.manager || isCreator(state);
```

Comment:
Logic seems to be flipped - curators aren't managers?

Reply:
Hmmm maybe you misread, that reads "manager or creator", not curator?

Reply:
Why does isManager return whether someone is "manager or creator"? isManager should only return true for managers, otherwise code blow would break.

Reply:
Whoops, creators :P Might need to take a dyslexia test